Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -15,13 +15,23 @@ protocol TabTrayController: UIViewController,
 }
 
 class TabTrayViewController: UIViewController, TabTrayController {
+    struct UX {
+        struct NavigationMenu {
+            static let height: CGFloat = 32
+            static let width: CGFloat = 343
+        }
+    }
+
     var themeManager: ThemeManager
     var themeObserver: NSObjectProtocol?
     var notificationCenter: NotificationProtocol
 
     var openInNewTab: ((URL, Bool) -> Void)?
     var didSelectUrl: ((URL, Storage.VisitType) -> Void)?
 
+    var layout: LegacyTabTrayViewModel.Layout = .compact
```

Comment:
This is in new code right? Should we use `LegacyTabTrayViewModel`? Or recreate a new one?

Reply:
Because we are going to use Redux we will not have ViewModel in this case so I'm putting all of this `state` elements in the viewController for the moment until I start the Redux tasks

Reply:
Oh ok 👍